pkg/kubeapiserver/options/plugins_test.go

```diff
@@ -22,9 +22,9 @@ import (
 )
 
 func TestAdmissionPluginOrder(t *testing.T) {
-	// Ensure the last four admission plugins listed are webhooks, quota, and deny
+	// Ensure the last four admission plugins listed are policies, webhooks, quota, and deny
```

Comment:
This doesn't look right. The order is correct, but what we say test no longer holds. We're now checking for mutating before validating and, within that, policy before external callout(s).

Reply:
Good point. I've rephrased.